net-firewall/shorewall/files/4.6/shorewall-lite.systemd-r1

```diff
@@ -0,0 +1,20 @@
+#
+#	The Shoreline Firewall Lite (Shorewall-Lite) Packet Filtering Firewall - V4.6
+#
+[Unit]
+Description=Shorewall IPv4 firewall lite
+Documentation=man:shorewall-lite(8) http://www.shorewall.net/Documentation_Index.html
+Wants=network-online.target
+After=network-online.target
+Conflicts=iptables.service firewalld.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+EnvironmentFile=/etc/conf.d/shorewall-lite
```

Comment:
This is against the [policy](https://wiki.gentoo.org/wiki/Project:Systemd/Ebuild_policy#Unit_file_guidelines) (pt 4), and I'm pretty sure repoman complains for that (you know you ought to `repoman full`, right? :)).